### PR TITLE
Webhook

### PR DESCRIPTION
**Pull request by luxflux**
This adds a simple webhook system.
The only event at the moment is triggered when an invoice is being paid.

### CODE REVIEW

**Comment by luxflux** [comment]
IMHO this is complete, could you have a look, @huerlisi?

File: db/schema.rb [hunk 1]
```diff
@@ -203,6 +203,12 @@
     t.decimal  "discount",        :precision => 10, :scale => 2, :default => 0.0
   end
 
+  create_table "debit_direct_files", :force => true do |t|
```
Comment (huerlisi) [inline db/schema.rb]: Please cleanup the `schema.rb` file so it only contains changes in this PR.

Comment (luxflux) [inline db/schema.rb]: Good point! I really missed these changes... Could you have a look again, @huerlisi?

**Comment by huerlisi** [comment]
Looks good :+1:

**Pull request merged**